### ISSUE TITLE
GREAT - predicts functions of cis-regulatory regions

### ISSUE THREAD

**Issue by smithcr** [bgruening/galaxytools#184]
http://bejerano.stanford.edu/great/public/html/
Many coding genes are well annotated with their biological functions. Non-coding regions typically lack such annotation. GREAT assigns biological meaning to a set of non-coding genomic regions by analyzing the annotations of the nearby genes. Thus, it is particularly useful in studying cis functions of sets of non-coding genomic regions

**Comment by blankenberg** [comment]
For both a source of getting data into Galaxy and as a destination for sending data from Galaxy?

**Comment by bgruening** [comment]
@blankenberg yes ... this was the plan. But starting with a data source plugin.

**Comment by blankenberg** [comment]
Cool, let me know if you need help. 

FWIW, UCSC table browser is able to export to GREAT, so there is probably some sort of mechanism in place on their end that can be co-opted/used for sending genomic regions there.
